tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "compileOnSave": false,
+  "angularCompilerOptions": {
+    "enableIvy": false
```

Comment:
Ivy might be a good solution

Reply:
Ivy was not well supported in NX and was discouraged. How does it look like now?
With Ivy enabled we have a big warn in a console that as a library Ivy should be disabled.

Reply:
<img width="782" alt="Screenshot 2020-06-04 at 09 33 42" src="https://user-images.githubusercontent.com/21977168/83728200-7c352280-a646-11ea-9b30-762093ee7b07.png">

Reply:
https://github.com/nrwl/nx/issues/1364
I thnk we could try it now!
But maybe not in this lib, let's try it in an NX project first, to make sure everything is ok